### PR TITLE
ec2_import_image - new module

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

ec2_import_image - new module

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by tremble** [comment]
(rebased since my PR #1745 broke this one)

File: plugins/modules/ec2_import_image.py [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+        task_name=dict(type="str", aliases=["name"]),
+    )
+
+    module = AnsibleAWSModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "absent", ["task_name"]],
+            ["state", "present", ["task_name"]],
+        ],
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply plugins/modules/ec2_import_image.py]: Right! Removed required_if and added directly required=true. Thanks.

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

File: plugins/modules/ec2_import_image.py [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+            type: float
+        upload_start:
+            description:
+            - The time that the disk upload starts.
+            type: str
+  description:
+    description:
+      - A description string for the import image task.
+    type: str
+  disk_containers:
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply plugins/modules/ec2_import_image.py:60]: Done!

**Review by hakbailey — APPROVED** [review]

**Review by abikouo — APPROVED** [review]

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
regate

**Pull request merged**